### PR TITLE
Refactor: Refactored the files and folder using absolute path

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Refactor files and folder. 
Used absolute path

live preview: https://fabelle-learn-react.netlify.app/

### CODE REVIEW

File: src/components/pages/home/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            src="./images/homepage/home.svg"
+            alt="landing-img"
+          />
+        </div>
+      </section>
+
+      <section className="popular-courses">
+        <h2>Popular Courses</h2>
+        <div className="courses-container">
+          <Card
```
Comment (nayyyhaa) [inline src/components/pages/home/Home.jsx:28]: you can loop around using map here.

**Review by nayyyhaa — APPROVED** [review]
Rest of the code looks good, only a few exports, path changes(using index file) are there. you can merge it! 👍

File: src/components/pages/home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  return (
+    <main>
+      <section className="landing-page">
+        <div className="content">
+          <h1>Learning that gets you</h1>
+          <p>Skills for your present (and your future). Get started with us.</p>
+        </div>
+        <div className="represent-img">
+          <img
+            loading="lazy"
```
Comment (Developers-codz) [inline src/components/pages/home/Home.jsx:18]: if this image is on your landing page then adding `loading="lazy" ` is not good.

**Review by Developers-codz — APPROVED** [review]
Looks good to merge, just also map the cards as Neha suggested :)

**Pull request merged**